libcontainer/dmz/nolibc/arch-aarch64.h

```diff
@@ -0,0 +1,157 @@
+/* SPDX-License-Identifier: LGPL-2.1 OR MIT */
+/*
+ * AARCH64 specific definitions for NOLIBC
+ * Copyright (C) 2017-2022 Willy Tarreau <w@1wt.eu>
```

Comment:
It says it can be MIT, which should be compatible with Apache-2.

See here: https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project

Am I missing something?

Reply:
Yes. I think we can just keep MIT and remove the LGPL-2.1. Just remove the confusion

Reply:
Idem as the other comment. I think I prefer to keep it if we can keep it. Otherwise, updating to a newer version of nolibc in the future would be more painful (or we need to make a script that nolibc, removes this, maybe something else, etc.).

I think there is value in the simplicity of just a simple c&p, no patches added.

Reply:
> I think there is value in the simplicity of just a simple c&p, no patches added.

The problem is that this is legally less simple.
It is not obvious (for non-maintainers) that the GPL-2.0 code is not linked with runc.

Reply:
@cyphar thanks for the comment!